src/ifElse.test.js

```diff
@@ -1,11 +1,46 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
+  let firstCb;
+  let secondCb;
+  let condition;
```

Comment:
Please provide good variables names) instead of Cb write Callback)

Reply:
fixed